rel/files/app.config

```diff
@@ -1,4 +1,34 @@
 [
+ {kernel, [
+  {logger_level, warning},
+  {logger, [
+    %% Console logger
+    {handler, default, logger_std_h, #{}},
+    %% Disk logger for errors
+    {handler, disk_log, logger_disk_log_h,
+       #{config => #{
+           file => "{{mongooseim_log_dir}}/mongooseim.log",
+           type => wrap,
+           max_no_files => 5,
+           max_no_bytes => 2097152,
+           sync_mode_qlen => 2000, % If sync_mode_qlen is set to the same value as drop_mode_qlen,
+           drop_mode_qlen => 2000, % synchronous mode is disabled. That is, the handler always runs
+           flush_qlen => 5000,     % in asynchronous mode, unless dropping or flushing is invoked.
+           burst_limit_enable => true, % Default
+           burst_limit_max_count => 20, % Default
+           burst_limit_window_time => 500, % Default
+           overload_kill_enable => true,
+           overload_kill_qlen => 20000, % Default
+           overload_kill_mem_size => 3000000, % Default
+           overload_kill_restart_after => 5000 % Default
```

Comment:
Why do we need to explicitly set the default values?

Reply:
I thought so for readability, so that we know the exact expected behaviour of the logging system. I could also just leave a link to the documentation now that I think about it 🤔